packages/mars/src/options/chain/arbitrum.ts

```diff
@@ -0,0 +1,21 @@
+import { Chain } from './model'
+
+export const arbitrum: Chain = {
+  chainId: 42161,
+  chainName: 'Arbitrum',
+  getPublicRPC: () => 'https://arb1.arbitrum.io/rpc',
+  getInfuraRPC: (infuraApiKey) => `https://arbitrum-mainnet.infura.io/v3/${infuraApiKey}`,
+  getAlchemyRPC: (alchemyApiKey) => `https://arb-mainnet.g.alchemy.com/v2/${alchemyApiKey}`,
+  getBlockExplorerContractAddress: (contractAddress) => `https://arbiscan.io/address/${contractAddress}`,
+  getEtherscanVerifierApi: () => 'https://api.arbiscan.io/api',
+}
+
+export const arbitrum_rinkeby: Chain = {
+  chainId: 421611,
+  chainName: 'Arbitrum Testnet',
+  getPublicRPC: () => 'https://rinkeby.arbitrum.io/rpc',
+  getInfuraRPC: (infuraApiKey) => `https://arbitrum-rinkeby.infura.io/v3/${infuraApiKey}`,
```

Comment:
Quite some duplication for infura/alchemy specific URI building? can we have a private fn (apiKey, name) => {} and call like:
```
getInfuraRPC: (infuraApiKey) => getInfuraRpc(infuraApiKey, 'arbitrum-rinkeby')
```
and similar for alchemy and others?